### PR TITLE
mostly changed buttons

### PR DESCRIPTION
**Pull request by Raeludwig**

### CODE REVIEW

File: frontend/src/components/shared/ResourceCard.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+const ResourceCard = ({ image, title, description, link, bg, buttonStyle, buttonText }) => {
   const cardClass = bg ? 'card-with-bg' : 'card-without-bg';
 
   return (
     <div className={`resource-card ${cardClass}`}>
       <img src={image} alt={title} />
       <h2>{title}</h2>
       <p>{description}</p>
-      <a href={link} className="learn-more-link">Learn More</a>
+      <button onClick={() => window.location.href = link} className={`learn-more-button ${buttonStyle}`}>
```
Comment (rose-codes) [inline frontend/src/components/shared/ResourceCard.js:12]: Screenshot 2023-10-19 at 2 29 23 PM
For some reason, the `learn-more-button` is throwing undefined and appears like this in the browser

File: frontend/src/pages/HomePage.js [hunk 2]
```diff
@@ -115,7 +116,7 @@ const HomePage = () => {
               >
                 We are silencing abuse
               </h1>
-              <div class="py-2 fs-5">
+              <div  class="py-2 fs-5">
```
Comment (rose-codes) [inline frontend/src/pages/HomePage.js:119]: Change `class` to `className`

**Review by rose-codes — COMMENTED** [review]
Looks good so far! Just a few typos to fix!

**Review by arzucaner — CHANGES_REQUESTED** [review]
You did a great job! You can make the necessary updates according to Rose's suggested changes and send them. Thank you, Rae!

**Pull request closed without merging**